psychopy/experiment/components/_base.py

```diff
@@ -498,6 +498,47 @@ def writeParamUpdate(self, buff, compName, paramName, val, updateType,
                 if stopVal in ['', None, -1, 'None']:
                     stopVal = '-1'
                 buff.writeIndented(f"{compName}.setSound({params['sound']}, secs={stopVal}){endStr}\n")
+            elif paramName == 'movie':
+                # we're going to do this for now ...
+                if params['units'].val == 'from exp settings':
+                    unitsStr = "units=''"
+                else:
+                    unitsStr = "units=%(units)s" % params
+
+                if params['backend'].val == 'moviepy':
+                    code = ("%s = visual.MovieStim3(\n" % params['name'] +
+                            "    win=win, name='%s', %s,\n" % (
```

Comment:
In general I'd advise against including indentation in the string - it's better to use `buff.setIndentLevel(+1, relative=True)` (we've done manual indentation in loads of places elsewhere but it's bad and someday I wanna do a sweep to get rid of it all)